datafusion/core/tests/sql/group_by.rs

```diff
@@ -654,13 +654,13 @@ async fn group_by_dictionary() {
         .expect("ran plan correctly");
 
         let expected = vec![
-            "+-----+------------------------+",
-            "| val | COUNT(DISTINCT t.dict) |",
-            "+-----+------------------------+",
-            "| 1   | 2                      |",
-            "| 2   | 2                      |",
-            "| 4   | 1                      |",
-            "+-----+------------------------+",
+            "+-------+------------------------+",
```

Comment:
I only fixed this for the `single_distinct_to_groupby` case but it would be good to fix this consistently for all cases. I will see if I can fix in this PR, or file a new issue if it not trivial

Reply:
Actually. I'm confused now as well ... perhaps this name should not have been qualified in the first place? I'll take another look

Reply:
I understand this now. The test above is referencing  a column named `t.val` and the physical plan drops column qualifiers:

```
Column::new(&c.name, idx))
```

The updated test is referencing a column with an alias of `t.val,` and the physical plan uses the alias name, hence the different output.